images/mongo/Dockerfile

```
@@ -1,6 +1,6 @@
 ARG IMAGE_REPO
 FROM ${IMAGE_REPO:-lagoon}/commons as commons
-FROM alpine:3.8
+FROM alpine:3.9
```

```suggestion
FROM alpine:3.8
```
Reverting back to 3.8 to avoid upgrades to images as part of this process.